script/generate_matrix.sh

```diff
@@ -0,0 +1,29 @@
+find . -type f -name "*.feature" > features.txt
```

Comment:
There's a part of me is slightly leary of hand crafting JSON via bash, but it is a simple use, and means we avoid dependencies within the environment, and whilst not escaping the names, it does quote them which is probably sufficient.

Reply:
This is what is implemented in CD.